thrimshim/thrimshim/main.py

```diff
@@ -73,11 +75,40 @@ def get_row(ident):
 	}
 	return json.dumps(response)
 
+def get_all_rows():
+	"""Gets all rows from the events table from the database"""
+	conn = app.db_manager.get_conn()
+	results = database.query(conn, """
+		SELECT *
+		FROM events""")
+	
+	rows = []
+	row = results.fetchone()
+	while row:
+		row = row._asdict()
+		row['id'] = str(row['id'])
+		row = {
+			key: (
+				value.isoformat() if isinstance(value, datetime.datetime)
+				else value
+			) for key, value in row.items()
+		}
+		rows.append(row)
+		row = results.fetchone()
+	if not rows:
+		return 'No rows found', 404
```

Comment:
If there's no rows, I'd rather return 200 and an empty list, ie. don't special case it.